modules/ding_borgun_checkout/ding_borgun_checkout.admin.inc

```diff
@@ -76,6 +76,20 @@ function ding_borgun_checkout_settings($form, $form_state) {
 
   $form['env']['#validate'] = array('ding_borgun_checkout_settings_validate');
 
+  $form['allowed_pages'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Allowed pages'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+  );
+
+  $form['allowed_pages']['ding_borgun_checkout_allowed_pages'] = array(
+    '#type' => 'textarea',
+    '#title' => t('Allowed pages'),
+    '#description' => t("Comma separated list of URL's that are allowed to access with redirecting"),
```

Comment:
The convention is that it's newline separated.

And it's not URLs, it's paths.

The description should be something more like "List of pages allowed to redirect to payment page."

Reply:
True. I'll fix it. :) 
Thanks.